### PR TITLE
Solved the issue Logscale plotting uses incorrect x range for adaptive=False #25190

### PR DESCRIPTION
**Pull request by abhishek-kuma**
### Changed the argument of logspace in plotting.py
Solves the issue Logscale plotting uses incorrect x range for adaptive=False #25190 
#### Brief description of what is fixed or changed
```list_x = np.logspace(np.log10(int(self.start)), np.log10(int(self.end)),```
```list_x = np.logspace(np.log10(self.start), np.log10(self.end), num=self.nb_of_points)```
Arguments of logspace is changed so that it matches when we try to plot in logscale arguments
#### Other comments
In mailmap my email address and name is added.
### Release notes

* plotting
  * solves the issue in the function `_uniform_sampling'
  * added test to the function 'test_issue_25190()`

### CODE REVIEW

**Comment by abhishek-kuma** [comment]
@1e9abhi1e10  Please check this PR I've added tests to it .

**Comment by oscarbenjamin** [comment]
The tests have failed so either the tests need to be changed or the code should be changed.

It is not necessary to close the PR and open a new one to make changes. You can push further commits to the same PR.

**Comment by abhishek-kuma** [comment]
Only one ` Required ` case was not passed that is ` test / py2-import (pull_request) ` test and it mentions ` Warning: The support for python 2.7 will be removed on June 19. Related issue: https://github.com/actions/setup-python/issues/672
  Version 2.7 was not found in the local cache
  Error: The version '2.7' with architecture 'x64' was not found for Ubuntu 20.04.
  The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json ` and other two were tests which were not Required and also donot belong to the module I committed they are from ` sympy/core/sympify.py:495: in sympify ` and ` praising ` .

**Comment by oscarbenjamin** [comment]
> Version 2.7 was not found in the local cache Error:

Okay, that is an unrelated problem.

File: .mailmap [hunk 1]
```diff
@@ -213,6 +214,7 @@ Abhinav Cillanki <abhinavcillanki@kgpian.iitkgp.ac.in> JebronLames32 <abhinavcil
 Abhishek <uchiha@pop-os.localdomain>
 Abhishek Chaudhary <ac5003@columbia.edu>
 Abhishek Garg <abhishekgarg119@gmail.com>
+ABHISHEK KUMAR <96915839+abhishek-kuma@users.noreply.github.com> ABHISHEK KUMAR <abhishekkumaruprama1@gmail.com>
```
Comment (1e9abhi1e10) [inline .mailmap:217]: It should be like this:
`ABHISHEK KUMAR <abhishekkumaruprama1@gmail.com> ABHISHEK KUMAR <96915839+abhishek-kuma@users.noreply.github.com>`

**Comment by sylee957** [comment]
This is already resolved on the master

**Pull request closed without merging**